src/app/components/reports/pages/event-summary/view/view.component.html

```diff
@@ -0,0 +1,543 @@
+<h2>
+  Acoustic Event Summary Report
+
+  <a
+    id="print-button"
+    class="ms-2"
+    ngbTooltip="Print this report"
+    (click)="openPrintModal()"
+  >
+    <fa-icon [icon]="['fas', 'print']"></fa-icon>
+  </a>
+</h2>
+
+<p id="generatedByLabel">
+  Generated by {{ currentUser.userName }}
+  {{ report.generatedDate ?? DateTime.now() | dateTime }}
```

Comment:
```suggestion
  {{ report.generatedDate | dateTime }}
```

Ensure the model normalizes this